cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -555,6 +557,19 @@ def create_checkerKey(self):
                 "Error: {}".format(err_msg if ret_val != 0 else self.EMPTY_OUTPUT_LIST))
         return str(json_out[0]['key'])
 
+    def get_ceph_dashboard_link(self):
+        cmd_json = {"prefix": "mgr services", "format": "json"}
+        ret_val, json_out, err_msg = self._common_cmd_json_gen(cmd_json)
+        # if there is an unsuccessful attempt,
+        if ret_val != 0 or len(json_out) == 0:
+            raise ExecutionFailureException(
+                "'mgr services' command failed.\n" +
+                "Error: {}".format(err_msg if ret_val != 0 else self.EMPTY_OUTPUT_LIST))
+        if not 'dashboard' in json_out:
+            raise ExecutionFailureException(
+                "The ceph dashboard link does not exist")
```

Comment:
As @anmolsachan has suggested, can you make sure we don't raise any error for users who haven't enabled `ceph-dashboard`.

Instead of an error, if `dashboard` is not enabled, remove the key `rook-ceph-dashboard-link` from the JSON output.

Reply:
Done